preprocessing/tabular_data/feature_selection_tabular_data.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<p>After pruning the unimportant features, use <code>storemagic</code> to persist all relevant variables so that they can be reused in our next sequel notebook, <a href="https://github.com/aws/amazon-sagemaker-examples/raw/409a053852978284afea16bfe557264d8f33ade4/preprocessing/tabular_data/training_model_on_tabular_data.ipynb" target="_blank">training_model_on_tabular_data.ipynb</a>, where we focus on model training.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>